src/api/public/apidocs-new/paths/staging_staging_workflow_project_staging_projects_staging_project_name_copy_project_name.yaml

```diff
@@ -0,0 +1,60 @@
+post:
+  summary: Copy a staging project
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/staging_workflow_project.yaml'
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: path
+      name: staging_project_copy_name
+      schema:
+        type: string
+      required: true
+      description: The name of the staging project's copy
+      example: "openSUSE:Factory:Staging:B"
+  responses:
+    '200':
+      description: Get the state of a staging project.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/staging/project.yaml'
+          example:
+            name: 'home:Iggy:Staging:A'
+            staged_requests:
+              count: 0
+            untracked_requests:
+              count: 0
+            obsolete_requests:
+              count: 0
+            missing_reviews:
+              count: 0
+            building_repositories:
+              count: 0
+            broken_packages:
+              count: 0
+            checks:
+              count: 0
+            missing_checks:
+              count: 0
+            history:
+              count: 0
+    '404':
+      description: Not Found.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            not_found:
+              summary: Staging Workflow Project Not Found
+              value:
+                code: not_found
+                summary: 'Project home:Adminn not found.'
+            staging_project_not_found:
+              summary: Staging Project Name Not Found
+              value:
+                code: staging_project_not_found
+                details: 'Staging Project "home:Admin:Staging:AA" does not exist.'
```

Comment:
I think this is never returned. The copy controller action simply sets a delayed job. If the delayed job ends up realizing the staging project to be copied doesn't exist, that will never be reflected in the API result.

Reply:
Right